runtime/gc_vlhgc/ConfigurationIncrementalGenerational.cpp

```diff
@@ -287,6 +287,9 @@ MM_ConfigurationIncrementalGenerational::initialize(MM_EnvironmentBase *env)
 #define DEFAULT_MAX_AGE_FOR_PGC_COUNT_BASED				24
 #define DEFAULT_MAX_NURSERY_AGE							1
 #define DEFAULT_MAX_AGE_FOR_ALLOCATION_BASED			5
+#define DEFAULT_PGC_RATIO_FOR_EDEN_EXPANSION			0.07
```

Comment:
in the light of future optimizations in PGC time, I'd like to optimistically reduce this to 6%